README.md

```diff
@@ -2,55 +2,158 @@
+```
+
+The result in this case is of course `false`, because 7 and 12 are not equal numbers. We can also determine if two values are **not** equal, like this:
+
+```swift
+let compareApplesToOranges = numberOfApples != numberOfOranges // true, because 7 is indeed not equal to 12
+```
+
+In this case, the result is `true`. We may also be interested in learning if we have more apples than oranges, for this we have operators for greater and less than:
+
+```swift
+let moreApplesThanOranges = numberOfApples > numberOfOranges // false, because 7 is not greater than 12
+let lessApplesThanOranges = numberOfApples < numberOfOranges // true, because 7 is less than 12
+```
+
+Finally, there are also comparison operators for greater and less than or equal. Those will either be true if the two operands are equal or if the left hand side is greater, or respectively less, than the right hand side.
+
+```swift
+let lessOrTheSameNumberOfApplesAndOranges = numberOfApples <= numberOfOranges // true, because 7 is less than 12
+
+numberOfApples = 12
+let moreOrTheSameNumberOfApplesAndOranges = numberOfApples >= numberOfOranges // true, because 12 is not greater than 12, but 12 is equal to 12
+```
+
+These comparisons might not seem too useful on their own, but you will learn later how you can use them in *conditional expressions* to execute different statements in your programs, depending on the value of these kind of conditions.
+
+## Logical operations
+
+In the last example, we already saw a simple logical statement for our program, we were able to determine if two values are greater than or equal to each other. Boolean values can also be combined using *logical operators* to formulate more complex logical statements about variables. 
+
+The logical `NOT` operator inverts the value of a Boolean, so that `true` becomes `false` and vice versa. This operator is a *unary* operator, because it operates on a single value, in contrast to the *binary* comparison operators, which operate on two values. The operator is put in front of the operand, without any whitespace between them:
+
+```swift
+let asManyApplesAsOranges = false
+let notAsManyApplesAsOranges = !asManyApplesAsOranges // true
+```
+
+It can be read as *not* `asManyApplesAsOranges` to make its use as a prefix clearer. If we choose our variable names carefully, we can ensure that such logical expressions can be read like this and make our code more readable in that way.
+
+The logical `AND` operator forms a logical expression which is `true` if both values are `true`.
 
 ```swift
 let isTheSkyBlue = true
 let isTheGrassGreen = true
 let canHumansFly = false
 let canDolphinsWalk = false
+
+let isTheSkyBlueAndTheGrassGreen = isTheSkyBlue && isTheGrassGreen // true, because both values are true
+let canHumansFlyAndIsTheSkyBlue = canHumansFly && isTheSkyBlue // false, because canHumansFly is false
+let canHumansFlyAndDolphinsWalk = canHumansFly && canDolphinsWalk // false, because both values are false
 ```
```

Comment:
I would prefer we take out the variables here and go with just the following:

```swift
isTheSkyBlue && isTheGrassGreen // true, because both values are true
canHumansFly && isTheSkyBlue // false, because canHumansFly is false
canHumansFly && canDolphinsWalk // false, because both values are false
```